.editorconfig

```diff
@@ -19,3 +19,7 @@ max_line_length = 99
 [{**.yml, **.yaml, **.jinja}]
 indent_style = space
 indent_size = 4
+
+[{CMakeLists.txt, **.cmake}]
+indent_style = space
+indent_size = 4
```

Comment:
This project seems to use 2 spaces in CMake files, at least that was my experince when working with the CMake files.